### PR TITLE
Update storing and bootstrapping data

### PR DESCRIPTION
**Pull request by joshkadis**
1. Store chart type as string in its own meta field
2. Send data back as a single postMessage instead of one per data object

### CODE REVIEW

File: js/plugin/post-edit.js [hunk 1]
```diff
@@ -100,38 +100,35 @@ function WPSimplechartApp( $ ) {
...
 			} catch( e ) {
 				toSend = window.WPSimplechartBootstrap[ key ];
 			}
-			toWindow.postMessage( {
-				data: toSend,
-				messageType: 'bootstrap.' + key
-			}, '*' );
-		}
+			data[ key ] = toSend;
+			return data;
```
Comment (zgreen) [inline js/plugin/post-edit.js:130]: Could one-liner this:

``` js
return { ...data, [key]: toSend };
```

Comment (joshkadis) [reply js/plugin/post-edit.js:130]: WP standards here

**Comment by zgreen** [comment]
🍕

File: js/plugin/post-edit.js (hunk 1 above)
Comment (zgreen) [reply js/plugin/post-edit.js:130]: right!

**Pull request merged**